src/components/Accordion/index.js

```diff
@@ -0,0 +1,106 @@
+import React, { PureComponent } from "react";
+import { bool, node } from "prop-types";
+import styled from "styled-components";
+
+import Section from "./Section";
+import Panel from "./Panel";
+
+const Wrapper = styled.div`
+  > :nth-child(n + 1)::after {
+    background: #ebebeb;
+    content: " ";
+    display: flex;
+    height: 1px;
+    margin-left: 0px;
+    margin-right: 0px;
+  }
+  > :nth-last-child(1)::after {
+    background: none;
+    content: " ";
+    display: flex;
+    height: 1px;
+    margin-left: 16px;
+    margin-right: 16px;
+  }
+`;
+
+class Accordion extends PureComponent {
+  static propTypes = {
+    children: node.isRequired,
+    multiToggle: bool
+  };
+
+  static defaultProps = {
+    multiToggle: false
+  };
+
+  constructor(props) {
+    super(props);
+    const openSections = {};
+
+    this.state = { openSections };
```

Comment:
I think we should add a prop to provide an initial state in case we want the accordion to be open by default

Reply:
Pretty good idea.